app/controllers/api/v1/scholarships_controller.rb

```diff
@@ -0,0 +1,13 @@
+module Api
+  module V1
+    class ScholarshipsController < ApplicationController
+      def index
+        render json: Scholarship.all
+      end
+
+      def show
+        render json: Scholarship.find(params[:id])
```

Comment:
The `find` method can error out, if it does not find the record, for whatever reason.  To avoid that, one solution would be to use the `find_by` method.  i.e. `Scholarship.find_by(id: params[:id])`.  

If it does not find a record that meets the criteria, it just returns `nil`.  Give these a try in the rails console.

That said, you might want to pair this with some error protection (i.e. `if it finds the record, render it.  else, render some error sentence`).